internals/api/uuid/uuid.go

```diff
@@ -12,14 +14,17 @@ var (
 	ErrInvalidUUID = errio.Namespace("uuid").Code("invalid").ErrorPref("invalid uuid: %s")
 )
 
-// UUID is a wrapper around go.uuid.UUID
+// UUID is a wrapper around github.com/gofrs/uuid.UUID.
 type UUID struct {
 	gid.UUID
 }
 
 // New generates a new UUID.
 func New() UUID {
-	id := gid.NewV4()
+	id, err := gid.NewV4()
+	if err != nil {
+		panic(err)
```

Comment:
When does this error occur? Is it acceptable to `panic` in that case?

Reply:
This error occurs if reading from `rand.Reader` fails. It is acceptable to `panic`, in my opinion, because the previous `uuid` generator also panicked in this situation.